hooks/persistence-defectdojo/.helm-docs.gotmpl

```diff
@@ -130,7 +130,7 @@ can add these via annotation to the scan. See examples below.
 | `defectdojo.securecodebox.io/engagement-deduplicate-on-engagement` | Deduplicate On Engagement  | false                                                                | Only used when creating the Engagement not used for updating                          |
 | `defectdojo.securecodebox.io/engagement-tags`                      | Engagement Tags            | Nothing                                                              | Only used when creating the Engagement not used for updating                          |
 | `defectdojo.securecodebox.io/test-title`                           | Test Title                 | Scan Name                                                            |                                                                                       |
-
+| `defectdojo.securecodebox.io/minimum_severity`                     | Minimum severity for findings created in DD    | Nothing                                          | Used to only create finding in DD, which are of a certain severity                    |
```

Comment:
Why is the name with underscore and the others above with hyphen? This should be consistent.

Reply:
This was changed in commit 3999698c76a710f663a695acd90eb39842b36ff4